WordPress/Classes/ViewRelated/Reader/Detail/ReaderDetailViewController.swift

```diff
@@ -230,8 +234,16 @@ class ReaderDetailViewController: UIViewController, ReaderDetailView {
 
         if post.content?.hasSuffix("[…]") == true {
             let viewMoreView = ReaderReadMoreView(post: post)
-            webView.addSubview(viewMoreView)
-            viewMoreView.pinEdges([.horizontal, .bottom])
+            // Add to the scroll view's parent view instead of directly to webView
+            if let containerView = webView.superview {
+                containerView.addSubview(viewMoreView)
+                viewMoreView.translatesAutoresizingMaskIntoConstraints = false
+                NSLayoutConstraint.activate([
+                    viewMoreView.leadingAnchor.constraint(equalTo: webView.leadingAnchor),
+                    viewMoreView.trailingAnchor.constraint(equalTo: webView.trailingAnchor),
+                    viewMoreView.bottomAnchor.constraint(equalTo: webView.bottomAnchor)
```

Comment:
Should this code also include `topAnchor`? What `viewMoreView.frame.oring.y` is going to be?